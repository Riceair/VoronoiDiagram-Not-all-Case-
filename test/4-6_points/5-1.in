5
123 456
213 478
11 590
234 77
99 68