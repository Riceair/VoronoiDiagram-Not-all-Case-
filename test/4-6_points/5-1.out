P 11 590
P 99 68
P 123 456
P 213 478
P 234 77
E -328805 -55111 -220.75 282.51
E -220.75 282.51 139.5 583.6
E -220.75 282.51 154 259.33
E 139.5 583.6 214.44 277.03
E 139.5 583.6 70672 127794
E 154 259.33 214.44 277.03
E 154 259.33 5836.5 -84977.5
E 214.44 277.03 252844.4 13507.03
